generate-protos.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
-
+# Script to generate the Protobuf classes from google/google-cloudevents
 set -e
 
-# Script to generate the Protobuf classes from google-cloudevents
-
+echo "~ START"
```

Comment:
Personally I really don't think all of the extra chatter in the output is useful. I like the output of tools like this to be brief and to the point - make it clear what's going on for anything that's going to take a significant amount of time, and when it's completed successfully, but that's all.

If anyone needs to debug *exactly* where things are going wrong, changing the script to use `set -ex` is almost always going to be more useful than the extra logs here.

Reply:
I think it's important to lightly document and describe what the program is doing.
It may be more obvious to the author how all of this works, but it's not obvious and clear to me. The program didn't work as expected and I feel like there isn't equal support for all 7 languages, hence the delay in this PR and getting things to work.

In the case of an error, I'd expect to see which step failed without a full `set -ex` to debug the script.